### PR TITLE
Added lombok and applied Getter-Setters to Models

### PR DESCRIPTION
**Pull request by ghost**
Hi Valentino,

I came across your repo. Seemed a good base to start my grasp on Springboot.

As suggested by you, i have added Lombok dependency in Pom.xml and also added Getters and Setters annotation to model.

Kindly review and let me know in case of any changes.

NOTE - This is my first Pull Request so guide me in case i miss anything :)

**WHAT**
1. Added Lombok dependency.
2. Added Getters and Setters to Models.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -92,6 +92,16 @@
             <version>${swagger.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.24</version>
+            <scope>provided</scope>
+        </dependency>
```
Comment (vpellegrino) [inline pom.xml:100]: I'm not sure that this dependency is actually needed.
Can we use directly the second one, with a fixed version?

**Review by vpellegrino — COMMENTED** [review]
@ninad0751 thanks for your PR.
And congratulations for your first contribution :rocket: 
I just left one question/observation about dep management

File: pom.xml (hunk 1 above)
Comment (ghost) [reply pom.xml:100]: I initially tried with the fixed version. But i was getting error and was unable to build it. I am able to run it with the version one. I have removed the second one(fixed version) from the PR as of now, but not sure if that will cause any issue in future. Please advice!
Error faced while building with fixed version dependency: https://stackoverflow.com/questions/66801256/java-lang-illegalaccesserror-class-lombok-javac-apt-lombokprocessor-cannot-acce

The above thread advised to switch to latest lombok version.
NOTE : I am using Java 17.0.1 version.

**Review by vpellegrino — APPROVED** [review]
LGTM, thank you for your contribution @ninad0751

**Pull request merged**